Doc/library/importlib.rst

```diff
@@ -1013,6 +1013,36 @@ find and load modules.
       :exc:`ImportError` is raised.
 
 
+.. class:: NamespacePath(name, path, path_finder)
+
+   Represents a :term:`namespace package`'s path (:attr:`module.__path__`).
+
+   When its ``__path__`` value is accessed it will be recomputed if necessary.
+   This keeps it in-sync with the global state (:attr:`sys.modules`).
+
+   The *name* argument is the name of the namespace module.
+
+   The *path* argument is the initial path value.
+
+   The *path_finder* argument is the callable used to recompute the path value.
+   The callable has the same signature as :meth:`MetaPathFinder.find_spec`.
```

Comment:
Note the warning about `MetaPathFinder.find_spec` not being found; probably requires the full name.


```suggestion
   The callable has the same signature as :meth:`importlib.abc.MetaPathFinder.find_spec`.
```